src/sentry/static/sentry/app/views/performance/vitalDetail/vitalChart.tsx

```diff
@@ -204,14 +229,20 @@ class VitalChart extends React.Component<Props> {
               includePrevious={false}
               yAxis={yAxis}
             >
-              {({results, errored, loading, reloading}) => {
+              {({results: _results, errored, loading, reloading}) => {
                 if (errored) {
                   return (
                     <ErrorPanel>
                       <IconWarning color="gray500" size="lg" />
                     </ErrorPanel>
                   );
                 }
+
+                const singleSeries = _results?.find(
+                  r => r.seriesName === `p75(${vitalName})`
+                );
+                const results = _results ? (singleSeries ? [singleSeries] : []) : []; // TODO: Fix this multiseries hack
```

Comment:
There is a function in utils/charts that lets you separate single/multi-series result shapes with typing.